Source/WebCore/PAL/pal/spi/cf/CFNetworkSPI.h

```diff
@@ -515,4 +516,49 @@ WTF_EXTERN_C_END
 @property (setter=_setPrivacyProxyFailClosedForUnreachableNonMainHosts:) BOOL _privacyProxyFailClosedForUnreachableNonMainHosts;
 @end
 
+namespace WebCore {
+
+inline NSURLCacheStoragePolicy toNSURLCacheStoragePolicy(CFURLCacheStoragePolicy policy)
```

Comment:
I don't think this is a good place for this.  An SPI header should be used for declaration of SPI, which this is not.

Reply:
Fixed by creating a `WebKit::CacheStoragePolicy` enum class.